packages/editor/src/plugins/jsx-blocks/YouTube.js

```diff
@@ -1,53 +1,14 @@
 /** @jsx jsx */
 import { jsx } from '@emotion/core'
-import React, { useState } from 'react'
 import Player from 'react-youtube'
-import { Flex } from 'theme-ui/layout'
 import isURL from 'is-url'
 import getYouTubeID from 'get-youtube-id'
 
-import Overlay from './Overlay'
-import Label from '../../components/Label'
-import Input from '../../components/Input'
-import Button from '../../components/Button'
-
-export const YouTubeForm = ({ value, onSubmit }) => {
-  const [state, setState] = useState({
-    videoId: value.videoId || ''
-  })
-  return (
-    <form
-      css={{
-        padding: 8
-      }}
-      onClick={e => {
-        e.stopPropagation()
-      }}
-      onSubmit={e => {
-        e.preventDefault()
-        onSubmit(state)
-      }}
-    >
-      <Flex flexWrap="wrap" alignItems="flex-end">
-        <Label mr={2}>
-          Video ID:
-          <Input
-            type="text"
-            name="videoId"
-            value={state.videoId}
-            onChange={e => {
-              const videoId = isURL(e.target.value)
-                ? getYouTubeID(e.target.value)
-                : e.target.value
-              setState({ ...state, videoId })
-            }}
-          />
-        </Label>
-        <Button>Apply</Button>
-      </Flex>
-    </form>
-  )
-}
+/* TODO
+const videoId = isURL(e.target.value)
```

Comment:
I think this could be handled with some sort of schema sanitizer kind of thing in the component's `propertyControls`

Reply:
Should we make this light wrapper a `@blocks/youtube` thing?

Reply:
Yeah, I'm keen to break stuff into packages once there's a clear pattern established and we (I) stop moving everything around. Think this is getting closer though